var/spack/repos/builtin/packages/py-isort/package.py

```diff
@@ -12,6 +12,13 @@ class PyIsort(PythonPackage):
     homepage = "https://github.com/timothycrosley/isort"
     url      = "https://pypi.io/packages/source/i/isort/isort-4.2.15.tar.gz"
 
+    version('4.3.20', sha256='c40744b6bc5162bbb39c1257fe298b7a393861d50978b565f3ccd9cb9de0182a')
     version('4.2.15', sha256='79f46172d3a4e2e53e7016e663cc7a8b538bec525c36675fcfd2767df30b3983')
 
-    depends_on('py-setuptools', type='build')
+    depends_on('py-setuptools', type=('build', 'run'))
```

Comment:
Are you sure this is needed at run time?

Reply:
Last question before merging this one.

Reply:
Yes, it is needed: isort's `main.py` explicitly imports `setuptools`